libs/gi/page-team/src/TeamCharacterSelector.tsx

```diff
@@ -63,16 +63,14 @@ export default function TeamCharacterSelector({
   const theme = useTheme()
   const isXs = useMediaQuery(theme.breakpoints.down('md'))
   const [editMode, setEditMode] = useState(false)
-  const [teamName, setTeamName] = useState(team.name)
-  const [teamDesc, setTeamDesc] = useState(team.description)
+  const teamName = useMemo(() => team.name, [team.name])
```

Comment:
In my opinion these two variables should not need to be memoized at all, and we should use `team.name` directly.